src/pages/editor/components/slate/slate-types.d.ts

```diff
@@ -77,6 +77,44 @@ export type PageElement = {
   padding: PaddingProps;
 }
 
+export type QuestionElement =
```

Comment:
I'm thinking `QuestionElement` should be structured differently, as it includes a question part/heading and an answer part.
So, `AnswerAreaElement` is not a valid alias, in fact `QuestionElement` as a whole should be renamed to something that represent a heading as I think that is what you wanna do. What do you think?